### PR TITLE
DDL Script

### PR DESCRIPTION
**Pull request by Kalid-Nuredin**
The DDL Script is written using Mysql,
contains 7 tables

### CODE REVIEW

File: Database/DDL.sql [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+-- Table structure for table `agent`
+--
+
+CREATE TABLE `agent` (
+  `agent_id` varchar(36) NOT NULL,
+  `nationality` varchar(50) NOT NULL,
+  `languages` varchar(100) NOT NULL,
+  `experience_since` datetime NOT NULL,
+  `contact_id` varchar(36) NOT NULL,
+  `admin_id` varchar(36) NOT NULL,
```
Comment (seidsid) [inline Database/DDL.sql]: you already have defined the relationships between agent and admin on line 61. So, you don't need this.

File: Database/DDL.sql [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `user`
+--
+
+CREATE TABLE `user` (
+  `user_id` varchar(36) NOT NULL,
+  `property_id` varchar(36) NOT NULL,
```
Comment (seidsid) [inline Database/DDL.sql]: You don't need this as well.. user and property has one to many relationship.. and you have already defined that in property line number 111.

File: Database/DDL.sql (hunk 1 above)
Comment (Kalid-Nuredin) [reply Database/DDL.sql]: Done

File: Database/DDL.sql (hunk 2 above)
Comment (Kalid-Nuredin) [reply Database/DDL.sql]: Done

**Review by seidsid — APPROVED** [review]

**Pull request merged**